lib/game.rb

```diff
@@ -0,0 +1,50 @@
+require '../lib/interface.rb'
+
+class Game
+  include Interface
+  attr_accessor :symbol
+
+  def initialize(board, player_1, player_2)
+    @board = board
+    @player_1 = player_1
+    @player_2 = player_2
+    @turn = 1
+   end
+
+  def win?(item)
```

Comment:
We can transfer this logic of determining whether a particular symbol ("X" or "O") is inside the winning_pattern. 

Another question that arose in my mind here is that, if we are keeping track of a `player` 's  positions then why are we not using those positions to know whether or not a player has won?

Still another question. Is this `win?` method used outside of the class? Or is this method just being used internally here? If so then we should just have put it under a `private` keyword